incognia_test.go

```diff
@@ -294,8 +294,8 @@ func (suite *IncogniaTestSuite) TestSuccessGetSignupAssessmentAfterTokenExpirati
 	suite.NoError(err)
 	suite.Equal(signupAssessmentFixture, response)
 
-	token, _ := suite.client.tokenManager.getToken()
-	token.ExpiresIn = 0
+	token, _ := suite.client.tokenProvider.GetToken()
+	token.(*accessToken).ExpiresIn = 0
```

Comment:
Since it's for testing and It's not on the assertion part of the code, more on the mocking part of the code, I think it's ok to leave it like that. What do you think?